modules/odc-connecting-components.adoc

```diff
@@ -68,29 +68,32 @@ This procedure walks through an example of creating a binding connection between
 * Ensure that you have installed the *Service Binding Operator* from *OperatorHub*.
 
 .Procedure
-. Create a backing `OperatorSource` that adds the PostgresSQL Operator provided by Red Hat to the *OperatorHub*. A backing `OperatorSource` exposes the binding information in secrets, ConfigMaps, status, and spec attributes.
-.. In the *Add* view, click the *YAML* option to see the *Import YAML* screen.
-.. Add the following YAML file to apply the `OperatorSource`:
+. Create a `CatalogSource` that adds the PostgresSQL Operator provided by Red Hat to the *OperatorHub*. A backing `Database` exposes the binding information in secrets, ConfigMaps, status, and spec attributes.
```

Comment:
@pmacik as per latest changes in the way CRDs/API objects are referenced in docs, the following changes will need to be made: 
Create a `CatalogSource` object/resource (use one of the two as relevant)....
A backing `Database` object/resource..
ConfigMaps/config maps